packages/cli/internal/pkg/cli/spec/project_test.go

```diff
@@ -122,6 +126,10 @@ schemaVersion: 0
 							{
 								Type:   "nextflow",
 								Engine: "nextflow",
+								ResourceRequirements: ResourceRequirement{
+									VCpus:          2,
```

Comment:
Lets add tests with partial configuration. e.g. either VCpus or MemoryLimitMiB is set, but not both.